src/main/scala/nl/knaw/dans/bag/Deposit.scala

```diff
@@ -336,6 +336,13 @@ class Deposit private(val baseDir: File,
     withDepositProperties(newProperties)
   }
 
+  /**
+   * Most methods in this library only manipulate the Bag and Deposit in memory.
+   * Saves the in-memory deposit to the file system. It saves all the bag files by calling `DansBag.save()`
```

Comment:
1) `Saves the in-memory deposit to the file system.` should be the first line. The rest is detail.
2) I don't hope you mean you keep all payload files in memory. At first sight `all the bag files` seems to suggest that. Please be precise in distinguishing payload files and tag files.